sky/tools/gn

```diff
@@ -48,7 +48,13 @@ def to_gn_args(args):
 
     gn_args['is_debug'] = args.debug
     gn_args['is_clang'] = args.clang and args.target_os not in ['android', 'fnl']
-    gn_args['flutter_develop_mode'] = args.develop
+
+    if args.develop:
+        gn_args['dart_runtime_mode'] = 'develop'
+    elif args.debug:
+        gn_args['dart_runtime_mode'] = 'profile'
```

Comment:
See https://codereview.chromium.org/1903583002/

Dart now builds in three modes (JIT, AOT, and AOT for deploy).  "profile" is an AOT mode and can be selected in GN with "--debug --deploy"

For now, the AOT modes on Android are selected with the --deploy flag.  Maybe it would make sense to rename this to --aot?